packages/framework/src/Resources/views/Admin/Content/TransportAndPayment/freeTransportAndPaymentLimitSetting.html.twig

```diff
@@ -35,12 +35,15 @@
                     {% set domainName = domain.getDomainConfigById(domainId).getName() %}
                     <tr id="{{ priceLimitForm.vars.id }}" class="js-free-transport-and-payment-price-limit" data-domain-id="{{ domainId }}">
                         {% if isMultidomain() %}
-                            <td>{{ domainName }}</td>
+                            <td>
```

Comment:
this was not necessary, at least it does not belong into this commit but never mind :slightly_smiling_face:

Reply:
accepted

Reply:
form_error block was added into free shipping and payment form page        

- to have the same error components in admin forms